template/private-npm-package/README.md

```diff
@@ -82,7 +82,7 @@ If you use the `Squash and merge` option on pull requests, take extra care to fo
 ### Releasing latest
 
 Commits to the `master` branch will be released with the `latest` tag,
-which is the default used when running `npm install` or `yarn install`.
+which is the default used when running `npm install` or `pnpm install`.
```

Comment:
Looks like Gutenberg supports pnpm, but we should try initialising a new package once we have a stable snapshot release to make sure that it works end to end.

Reply:
https://github.com/seek-oss/skuba/pull/1329#issuecomment-1859528370